airbyte-ci/connectors/pipelines/pipelines/models/contexts/pipeline_context.py

```diff
@@ -218,6 +218,13 @@ def dagger_cloud_url(self) -> Optional[str]:
     def remote_storage_enabled(self) -> bool:
         return self.is_ci and bool(self.ci_report_bucket) and bool(self.ci_gcp_credentials)
 
+    def _should_send_status_check(self) -> bool:
+        should_send = self.is_pr or any(
+            self.pipeline_name.startswith(override) for override in MANUAL_PIPELINE_STATUS_CHECK_OVERRIDE_PREFIXES
```

Comment:
@alafanechere I don't love this way of doing things but found that if I didn't override this here, I had to mark the regression test run as a PR. That created duplicated status checks: one called `Regression Tests` and one called `Regression Tests on <source>`. `Regression Tests on <source>` is what's created here, and has links to the report etc. so it's the only one I really want. If you have a suggestion of a nicer way to do this let me know.

I also found that `Regression Tests on <source>` wasn't getting sent unless the global status check argument was present, which I found surprising. Is that intentional?

Reply:
> That created duplicated status checks
This is intentional for usual connector testing: one global check (required) for all connectors being tested on the PR, and one for each connectors modified on the pr.
IMO you should make the `Regression Tests` one the required check (to ensure that it has been run on all modified connectors). `Regression Tests on <source>` could be just used to access reports. The tricky thing would be to mark `Regression Tests` pass even if it's not run on non certified connectors which could be modified in the same PR as a certified connector.

> I also found that Regression Tests on <source> wasn't getting sent unless the global status check argument was present, which I found surprising. Is that intentional?

I don't think it's intentional, did you get the reason why?